operators/operators_test.go

```diff
@@ -28,65 +28,81 @@ func TestOperators(t *testing.T) {
 	root := "./testdata"
 	files := [][]byte{}
 	if _, err := os.Stat(root); os.IsNotExist(err) {
-		t.Error("failed to find operator test files")
+		t.Fatal("failed to find operator test files")
 	}
+
 	if err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
 		if strings.HasSuffix(path, ".json") {
-			data, _ := os.ReadFile(path)
+			data, err := os.ReadFile(path)
+			if err != nil {
+				return err
+			}
 			files = append(files, data)
 		}
 		return nil
 	}); err != nil {
-		t.Errorf("failed to walk test files: %s", err.Error())
+		t.Fatalf("failed to walk test files: %s", err.Error())
+	}
+
+	captureMatrix := map[string]bool{
```

Comment:
This capture matrix in tests improves the coverage by also testing with/without capture in transactions.